bot/cogs/weather.py

```diff
@@ -0,0 +1,105 @@
+"""Commands for the weather module."""
+import discord
+from discord.ext import commands
+import aiohttp
+from bot.settings import WEATHER_TOKEN
+
+
+class Weather(commands.Cog, name='weather'):
+    """commands for the weather finder."""
+
+    def __init__(self, bot: discord.ext.commands.bot.Bot):
+        self.bot = bot
+
+    @commands.command(
+        brief="Takes in a city name and returns the weather for that location")
+    async def weather(self, ctx: discord.ext.commands.context.Context, *, args: str) -> None:
```

Comment:
This type hint import is wayyyy too long, you add the `context` import to the import that already exists, i.e
```python
from discord.ext import commands, context
```
then do this 
```suggestion
    async def weather(self, ctx: context.Context, *, args: str) -> None:
```